staker/block_validator.go

```diff
@@ -86,6 +88,9 @@ type BlockValidatorConfig struct {
 	PendingUpgradeModuleRoot string                        `koanf:"pending-upgrade-module-root"` // TODO(magic) requires StatelessBlockValidator recreation on hot reload
 	FailureIsFatal           bool                          `koanf:"failure-is-fatal" reload:"hot"`
 	Dangerous                BlockValidatorDangerousConfig `koanf:"dangerous"`
+	DataPoster               dataposter.DataPosterConfig   `koanf:"data-poster" reload:"hot"` // DO NOT SUBMIT: add default values before sending for a review.
+	RedisUrl                 string                        `koanf:"redis-url"`
```

Comment:
Need to add DataPoster, RedisUrl and RedisLock to BlockValidatorConfigAddOptions for cli.

Reply:
Done.